<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>287</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Particulas</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="7">
     <widget class="QSpinBox" name="origenY_spinBox">
      <property name="maximum">
       <number>500</number>
      </property>
     </widget>
    </item>
    <item row="9" column="0" colspan="11">
     <widget class="QPlainTextEdit" name="salida"/>
    </item>
    <item row="5" column="1">
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>X:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="2" colspan="3">
     <widget class="QSpinBox" name="velocidad_spinBox">
      <property name="maximum">
       <number>100</number>
      </property>
     </widget>
    </item>
    <item row="0" column="0" rowspan="2" colspan="2">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>ID:</string>
      </property>
     </widget>
    </item>
    <item row="8" column="4" colspan="4">
     <widget class="QPushButton" name="Final_pushButton">
      <property name="text">
       <string>AGREGAR FINAL</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0" colspan="2">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Color:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>X:</string>
      </property>
     </widget>
    </item>
    <item row="7" column="5">
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>G</string>
      </property>
     </widget>
    </item>
    <item row="8" column="8" colspan="3">
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>MOSTRAR</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2" colspan="2">
     <widget class="QSpinBox" name="origenX_spinBox">
      <property name="maximum">
       <number>500</number>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="4">
     <widget class="QLineEdit" name="ID_lineEdit"/>
    </item>
    <item row="8" column="0" colspan="4">
     <widget class="QPushButton" name="Inicio_pushButton">
      <property name="text">
       <string>AGREGAR INICIO</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="2">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Velocidad:</string>
      </property>
     </widget>
    </item>
    <item row="7" column="10">
     <widget class="QSpinBox" name="B_spinBox">
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
    </item>
    <item row="4" column="4" colspan="2">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>DESTINO</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4" colspan="2">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>ORIGEN</string>
      </property>
     </widget>
    </item>
    <item row="7" column="9">
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>B</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2" colspan="2">
     <widget class="QSpinBox" name="destinoX_spinBox">
      <property name="maximum">
       <number>500</number>
      </property>
     </widget>
    </item>
    <item row="3" column="5" colspan="2">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Y:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="7">
     <widget class="QSpinBox" name="destinoY_spinBox">
      <property name="maximum">
       <number>500</number>
      </property>
     </widget>
    </item>
    <item row="5" column="5" colspan="2">
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Y:</string>
      </property>
     </widget>
    </item>
    <item row="7" column="2">
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>R</string>
      </property>
     </widget>
    </item>
    <item row="7" column="6" colspan="2">
     <widget class="QSpinBox" name="G_spinBox">
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
    </item>
    <item row="7" column="3" colspan="2">
     <widget class="QSpinBox" name="R_spinBox">
      <property name="maximum">
       <number>255</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>287</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
